src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Locations/Queries/GetPostcodes/GetPostcodeQueryResult.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+
+namespace SFA.DAS.ApprenticeAan.Application.Locations.Queries.GetPostcodes;
+
+public class GetPostcodeQueryResult
+{
+    public CoordinatesItem Coordinates { get; set; } = new();
+
+    public static implicit operator GetPostcodeQueryResult(GetAddressesListItem addressesListItem)
+    {
+        if (addressesListItem == null || addressesListItem.Latitude == null || addressesListItem.Longitude == null) return null!;
```

Comment:
The lat and long null checks are redundant... this is already being filtered in the handler.

Reply:
Removed